src/app/app.module.ts

```diff
@@ -4,14 +4,16 @@ import { FormsModule } from '@angular/forms';
 import { AppComponent } from '../app/app.component/app.component';
 import { DashboardComponent } from '../app/dashboard.component/todo.dashboard';
 import { EditComponent } from '../app/edit.component/edit.component';
+import { TodDetailComponent } from '../app/stats.component';
```

Comment:
component name and its file name must relate to each other. TodDetailComponent with stats.component file make no sense.